pkg/testing/case.go

```diff
@@ -56,13 +56,14 @@ type Secure struct {
 
 // TestCase represents a test case
 type TestCase struct {
-	ID      string   `yaml:"id,omitempty" json:"id,omitempty"`
-	Name    string   `yaml:"name,omitempty" json:"name,omitempty"`
-	Group   string   `yaml:"group,omitempty" json:"group,omitempty"`
-	Before  *Job     `yaml:"before,omitempty" json:"before,omitempty"`
-	After   *Job     `yaml:"after,omitempty" json:"after,omitempty"`
-	Request Request  `yaml:"request" json:"request"`
-	Expect  Response `yaml:"expect,omitempty" json:"expect,omitempty"`
+	ID                 string   `yaml:"id,omitempty" json:"id,omitempty"`
+	Name               string   `yaml:"name,omitempty" json:"name,omitempty"`
+	Group              string   `yaml:"group,omitempty" json:"group,omitempty"`
+	Before             *Job     `yaml:"before,omitempty" json:"before,omitempty"`
+	After              *Job     `yaml:"after,omitempty" json:"after,omitempty"`
+	Request            Request  `yaml:"request" json:"request"`
+	Expect             Response `yaml:"expect,omitempty" json:"expect,omitempty"`
+	GrpcMaxRecvMsgSize int      `yaml:"grpc_max_recv_msg_size,omitempty" json:"grpc_max_recv_msg_size,omitempty"`
```

Comment:
I guess we don't need to add this field here. Because it is never used.

Reply:
I intend to utilize GrpcMaxRecvMsgSize to determine the size of the command line settings stored in the server type. Then, the testCase can establish a gRPC connection and set the appropriate receive size accordingly. However, I'm uncertain if this approach is overly complex or if there might be any potential issues.

Reply:
IMO, it might be not safe if allow users to set the limit for each request. For example, A give it a very big value, then send  too many requests. the server might let the server panic.

For now, I believe a global setting is enough.

Reply:
may I understand that setting restrictions on the command line isn't necessary, and can I directly establish a 4MB constant in pkg/server/remote_server.go to check if the response body is too large?

Reply:
I agree with adding a flag to the CLI to set the limit. I think it's not necessary to add it into `type TestCase struct {`. Others are ok to me.